projects/plugins/social/src/class-social-shares.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Class for the Social Shares.
+ *
+ * @package automattic/jetpack-social-plugin
+ */
+
+/**
+ * Register the Jetpack Social Shares Class.
+ */
+class Social_Shares {
+	const SOCIAL_SHARES_POST_META_KEY = '_publicize_shares';
+
+	/**
+	 * Return a list of of social shares.
+	 *
+	 * @param int $post_id The Post ID.
+	 *
+	 * @return array
+	 */
+	public static function get_social_shares( $post_id = null ) {
+		if ( empty( $post_id ) ) {
+			$post    = get_post();
+			$post_id = $post->ID ?? 0;
+		}
+
+		if ( empty( $post_id ) ) {
+			return array();
+		}
+
+		$shares = get_post_meta( $post_id, self::SOCIAL_SHARES_POST_META_KEY );
+
+		if ( empty( $shares ) ) {
+			return array();
+		}
+
+		$succesful_shares = array_filter(
+			$shares[0],
+			function ( $share ) {
+				return isset( $share['status'] ) && $share['status'] === 'success';
```

Comment:
Very minor nitpick. I wonder why Yoda is not enforced here, it surely is in the JS
```suggestion
				return isset( $share['status'] ) && 'success' === $share['status'];
```